coopaname_custom/models/hr_employee.py

```diff
@@ -81,15 +101,19 @@ def create_user_portal_access(self):
             if user:
                 user.write({"active": True, "groups_id": [(4, group_id)]})
             else:
-                new_partner_id = self.env["res.partner"].create(
+                partner_id = employee.address_home_id or self.env[
+                    "res.partner"
+                ].create(
```

Comment:
I find it a bit convoluted to use the `or` expression here. It's better in my opinion to test with the if's.

Reply:
I just discovered it and quite liked it as a short form, but I understand it can be more readable with ifs!

Reply:
```
Python 3.7.7 (default, May  6 2020, 16:39:25)
[Clang 11.0.3 (clang-1103.0.32.59)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import this
The Zen of Python, by Tim Peters

Beautiful is better than ugly.
Explicit is better than implicit.
Simple is better than complex.
Complex is better than complicated.
Flat is better than nested.
Sparse is better than dense.
**Readability counts.**
Special cases aren't special enough to break the rules.
Although practicality beats purity.
Errors should never pass silently.
Unless explicitly silenced.
In the face of ambiguity, refuse the temptation to guess.
There should be one-- and preferably only one --obvious way to do it.
Although that way may not be obvious at first unless you're Dutch.
Now is better than never.
Although never is often better than *right* now.
If the implementation is hard to explain, it's a bad idea.
If the implementation is easy to explain, it may be a good idea.
Namespaces are one honking great idea -- let's do more of those!
```